modules/repository/fork.go

```diff
@@ -12,6 +12,9 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 )
 
+// CanUserForkBetweenOwners returns true if user can fork between owners.
+// By default, a user can fork a repository from another owner, but not from themselves.
+// Many users really like to fork their own repositories, so add an experimental setting to allow this.
```

Comment:
```suggestion
// TODO: Many users really like to fork their own repositories, so add an experimental setting to allow this.
```

Reply:
But it is describing the fact, it is not a TODO

Reply:
Wait, we have this setting already?
Because as far as I'm aware, we have no such setting yet.
And without the TODO, it sounds like the setting exists already…

Reply:
Allow to fork repository into the same owner #32819 

![image](https://github.com/user-attachments/assets/e2b74b2b-62a3-4d58-a0ab-35fd39f6fd83)